src/App.tsx

```diff
@@ -1,73 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+import { Autocomplete } from './components/Autocomplete';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
```

Comment:
Your code is becoming professional with right usage of hooks and typescript. Here is a tip: always on selecting some data you can usa identifiers to make as selectable and make traffic between ids and not filled objects.

https://react.dev/learn/choosing-the-state-structure#avoid-duplication-in-state

Note: always think that when
1. You fetched a data from server
2. You want to use one of its data

So you can make reference for id and what is your selectedPerson will become more like

```js
const [selectedPersonId, setSelectedPersonId] = useState<string | null>(null)
const selectedPerson = data.find((person) => person.id === selectedPersonId)
```

Please read the docs I sent and you'll understand it better